### PR TITLE
fix size lenovo x240 touchpad

### PR DESCRIPTION
**Pull request by jeremy447**

### CODE REVIEW

File: hwdb.d/60-evdev.hwdb [hunk 1]
```diff
@@ -399,6 +399,14 @@ evdev:name:SynPS/2 Synaptics TouchPad:dmi:*svnLENOVO*:pvrThinkPad??40?:*
  EVDEV_ABS_35=::41
  EVDEV_ABS_36=::37
 
+# Lenovo X240 series
+evdev:name:SynPS/2 Synaptics TouchPad:dmi:*svnLENOVO*:pvrThinkPadX240:*
+evdev:name:SynPS/2 Synaptics TouchPad:dmi:*svnLENOVO*:pvrThinkPadX240?:*
```
Comment (keszybz) [inline hwdb.d/60-evdev.hwdb:404]: Hmm, the line above would also match. It was added in bbe7cdeb8fa6e880a1296e170648e468036e843b.

Comment (jeremy447) [reply hwdb.d/60-evdev.hwdb:404]: Indeed. I should have said it but I asked first to Peter Hutterer and he said that will be ok
https://gitlab.freedesktop.org/libinput/libinput/-/issues/521

Comment (keszybz) [reply hwdb.d/60-evdev.hwdb:404]: I'll merge this with an adjusted commit message.

**Comment by IjonFryderyk** [comment]
Sorry to bother. I'm trying to find a fix for my inaccurate and erratic x240 touchpad for a while now. And I'm excited that someone finally succeeded. Great work Paviluf and keszybz! When would it be released for Ubuntu 20.04? Can I implement it myself?

**Pull request merged**